www/nikd01/cv04/index.php

```diff
@@ -0,0 +1,87 @@
+<?php
+require_once './utils/utils.php';
+
+$name = $email = $password = $confirm_password = '';
+$filename = __DIR__ . '/database/users.db';
+$errors = [];
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+    $name = trim(filter_input(INPUT_POST, 'name', FILTER_SANITIZE_STRING));
```

Comment:
![image](https://github.com/nvbach91/4IZ278-2023-2024-LS/assets/20724910/fa73cf18-d9a6-4056-8b47-19553a7c68de)

Reply:
This issue should be resolved now